RELEASE/scripts/autoscend/auto_consume.ash

```diff
@@ -1429,6 +1434,78 @@ boolean auto_knapsackAutoConsume(string type, boolean simulate)
 	return true;
 }
 
+boolean auto_chewAdventures()
+{
+	//tries to chew a size 4 familiar spleen item for adventures
+
+	//these items are all out of standard except in Quantum Terrarium
+	if(!is_unrestricted($item[Unconscious Collective Dream Jar]) && !is_unrestricted($item[Powdered Gold]) && !is_unrestricted($item[beastly paste]))
+	{
+		//powdered gold is also restricted in G lover, but if all these are restricted this must be standard
+		//no point continuing if standard restrictions apply
+		return false;
+	}
+
+	//called to eat size 4 spleen items when adventures < 10 and we can't eat or drink anything
+
+	int oldSpleenUse = my_spleen_use();
+
+	//do we wait until the last adventure to chew spleen?
+	boolean waitUntilLastAdventure = false;
+	if(item_amount($item[stench jelly]) >= 2)
+	{
+		int spleenForStenchJelly = min(my_adventures(),item_amount($item[stench jelly]));
+		//wait if chewing 4 spleen would take away the chance to use 2 or more stench jellies
+		if (oldSpleenUse - spleenForStenchJelly <= 2)
+		{
+			waitUntilLastAdventure = true;
+		}
+	}
+	if(auto_havePillKeeper() && spleen_left() == 6)
+	{
+		//if you have pill keeper maybe two force noncombat is better than chewing for adventures?
+		waitUntilLastAdventure = true;
+	}
+	if(waitUntilLastAdventure && my_adventures() > 1+auto_advToReserve())
+	{
+		return false;
+	}
+
+	if(spleen_left() >= 4)
+	{
+		//first the ones without the level 4 requirement because they give more stats
+		foreach it in $items[Unconscious Collective Dream Jar, Grim Fairy Tale, Powdered Gold, Groose Grease]
+		{
+			if(item_amount(it) > 0)
+			{
+				if(autoChew(1, it))
+				{
+					break;
+				}
+			}
+		}
+		if(have_effect($effect[Just the Best Anapests])>0)
+		{
+			uneffect($effect[Just the Best Anapests]);
+		}
```

Comment:
Unnecessary, already handled in `resetState()`

Reply:
resolved by @quarklikeadork